core/templates/core/panel.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="flex flex-row nowrap flex-grow h-full overflow-auto">
+        <div id="sidebar" class="flex md:flex flex-col gap-y-3 fixed md:relative top-0 left-[-18rem] md:left-0 md:translate-x-0 min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto duration-300 md:duration-0 border-r border-gray-600">
+            <a href="/" class="mx-4 pt-4 text-lg font-bold hover:underline">Strona Główna</a>
+            {% for namespace, links in sidebar_links.items %}
+                <div class="flex flex-col divide-y divide-y-reverse divide-gray-600">
+                    <h3 class="mx-4 py-3 text-lg font-bold">{{ namespace }}</h3>
+
+                    {% for link in links %}
+                    <a class="flex items-center justify-between mx-4 my-2 h-12 max-w-full rounded-xl hover:bg-slate-600 text-lg"
+                       href="{{ link.url }}" {% if link.new_tab %} target="_blank" {% endif %}>
+                        <span><i class="fa-regular fa-{{ link.icon }} mx-5 text-xl"></i><span>{{ link.name }}</span></span>
+                        <i class="fa-solid mx-2 {% if link.new_tab %}fa-arrow-up-right-from-square{% endif %}"></i>
+                    </a>
+                    {% endfor %}
+                </div>
+            {% endfor %}
+        </div>
+        {% block panel_content %}
+        {% endblock %}
+    </div>
+{% endblock %}
+
+{% block head %}
+    <script type="module">
+        const menuElement = document.querySelector("#sidebar")
+        menuElement.addEventListener("click", (event) => {
+            event.stopPropagation()
+        })
+        document.addEventListener('click', () => {
+            menuElement.classList.remove("translate-x-72")
+        })
+        document.querySelector("#mobile-menu-btn").addEventListener('click', (event) => {
+            event.stopPropagation();
+            menuElement.focus();
+            menuElement.classList.toggle("translate-x-72");
```

Comment:
Maybe when it is opened make background darker? Because now it looks kinda confusing (because navbar icon doesn't have animation)